databag/docker-compose.yml

```
@@ -0,0 +1,17 @@
+version: "3.7"
+
+services:
+
+  app_proxy:
+    environment:
+      APP_HOST: databag_web_1
+      APP_PORT: 7777
```

```suggestion
      APP_PORT: 7000
```
as it is the port used inside the container exposed by docker image.